### PR TITLE
WD-7536 LXD manage page 

### PR DESCRIPTION
**Pull request by carkod**
## Done

Build u.com/lxd/manage page

## QA

- Check https://canonical-com-1129.demos.haus/lxd/manage against [copy docs](https://docs.google.com/document/d/1doGyKYSTOsX8dHvQ6W9W2U_pHQY6yfiq1bqSAJyredQ/edit) and [design](https://www.figma.com/file/ZU2ufsTJD9tEkXPqhhDMZz/U.com---LXD-(rebranding)?node-id=1%3A3&mode=dev)

- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7536

### CODE REVIEW

**Comment by carkod** [comment]
Blocked by https://github.com/canonical/canonical.com/pull/1127

File: templates/lxd/manage.html [hunk 1]
```diff
@@ -0,0 +1,340 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1doGyKYSTOsX8dHvQ6W9W2U_pHQY6yfiq1bqSAJyredQ/edit{% endblock meta_copydoc %}
+
+{% block title %}Enterprise-grade MongoDB solution{% endblock %}
```
Comment (akbarkz) [inline templates/lxd/manage.html:5]: @carkod please stop blindly copy-paste stuff

File: templates/lxd/manage.html [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+    </div>
+    <div class="col">
+      <p>An official LXD UI tool is now available as an experimental feature. The UI supports most of the functionalities surrounding managing instances. More features coming soon.</p>
+      <p>
+        <a href="https://ubuntu.com/blog/lxd_ui">More on the LXD UI and future progress &nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <div class="u-align-center u-vertical-center">
```
Comment (akbarkz) [inline templates/lxd/manage.html:94]:
```suggestion
    <div class="u-align--center u-vertically-center">
```
wrong class names

File: templates/lxd/manage.html [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--white is-deep">
+  <div class="row">
+    <div class="col-9 col-medium-4 col-start-large-4 col-start-medium-3">
+      <h2><a href="/lxd/contact-us" aria-controls="lxd-install-modal">Talk to our LXD experts&nbsp;&rsaquo;</a></h2>
+      </div>
```
Comment (akbarkz) [inline templates/lxd/manage.html:326]: that's a redundant closing div element

File: templates/lxd/manage.html [hunk 4]
```diff
@@ -0,0 +1,392 @@
...
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--white is-deep">
+  <div class="row">
+    <div class="col-9 col-medium-4 col-start-large-4 col-start-medium-3">
+      <h2><a href="/lxd/contact-us" aria-controls="lxd-install-modal">Talk to our LXD experts&nbsp;&rsaquo;</a></h2>
```
Comment (akbarkz) [inline templates/lxd/manage.html:378]: `/lxs/contact-us` doesn't exist, did you mean `/contact-us` as in `/lxd/install` page?

**Review by akbarkz — APPROVED** [review]

**Pull request merged**